src/setupWorker/start/utils/getWorkerByRegistration.ts

```diff
@@ -13,6 +16,9 @@ export const getWorkerByRegistration = (
   ]
   const existingStates = allStates.filter(Boolean) as ServiceWorker[]
   const mockWorker = existingStates.find((worker) => {
+    if (serviceWorkerFileMatcher) {
+      return serviceWorkerFileMatcher(worker.scriptURL)
+    }
     return worker.scriptURL === absoluteWorkerUrl
```

Comment:
2 Qs.  

1) feels like absoluteWorkerUrl should be passed to the predicate.
2) then you can instead have serviceWorkerFileMatcher default to `(a,b) => a === b` (with better names)

Reply:
Yea, I like this idea. I opened a PR into this one as an example for review here: https://github.com/msutkowski/msw/pull/1/files

Reply:
I think we could safely rename `serviceWorkerFileMatcher` -> `serviceWorkerMatcher`. Not sure on this one, as they both make sense to me.

Reply:
👍 I like both changes

Reply:
👍

Reply:
Thanks for the input everyone - I've updated this PR with these changes.